plasmapy/physics/quantum.py

```diff
@@ -1,14 +1,22 @@
 """
 Functions for quantum parameters, including electron degenerate
 gases and warm dense matter.
+
+# TODO:
+    -fix ion sphere radius to use ion density
```

Comment:
I am 99% sure I dealt with this. It had to do with using ion density instead of electron density in the Wigner Seitz radius function and where it was called. That required me to implement z_mean to do n_i = n_e / z_mean.

This is what I do under GMS methods for the impact_paramter function

```
n_i = n_e / z_mean
# mean ion sphere radius.
ionRadius = Wigner_Seitz_radius(n_i)
```

Though it looks like missed it for the coupling parameter function... will fix it now.

Reply:
Fixed.

Reply:
fixed